packages/api/apollo-server/types/index.ts

```diff
@@ -0,0 +1,63 @@
+import { gql } from 'apollo-server-core'
+
+const Types = gql`
+  type Query {
+    users: [User]
+  }
+
+  type Mutation {
+    userMutation: [User]
+  }
+
+  enum UserType {
+    TA
+    STUDENT
+    INSTRUCTOR
+  }
+
+  type User {
+    id: ID!
+    firstName: String
+    lastName: String
+    utorID: String!
+    email: String
+    userType: UserType!
+    lastLogin: String
+    isActive: Boolean
+  }
+
+  type Assignment {
+    id: ID!
+    name: String
+    requiredFiles: [String!]
+    peerReviewDeadline: String
+    feedbackQuestions: [String!]
+    groupSize: Int!
+    isActive: Boolean
+  }
+
+  type File {
+    id: ID!
+    name: String!
+    path: String!
+    annotations: [Annotation]
```

Comment:
Should this be `[Annotation!]!`?

Reply:
> Should this be `[Annotation!]!`?

It's for the case a file has not been reviewed by any users yet.

Reply:
If there's no annotations, wouldn't it just be an empty array?